charts/meep-platform-ctrl/values.yaml

```diff
@@ -25,6 +25,7 @@ image:
   pullPolicy: Always
   env:
     MEEP_SESSION_KEY: "my-secret-key"
+    MEEP_MAX_SESSIONS: "10"
```

Comment:
I guess this is OK to specify this limit here for now.
This value should be deployment specific and therefore defined elsewhere in the deployment configuration.

Reply:
This is the default value. It can/should be overridden @ deployment, just like the session key. This is done in the user/values override yaml file.